packages/cli/generation/remote-generation/remote-workspace-runner/src/createAndStartJob.ts

```diff
@@ -148,6 +149,21 @@ async function createJob({
                 RelativeFilePath.of(PROJECT_CONFIG_FILENAME)
             );
             await writeFile(absolutePathToFernConfigJson, JSON.stringify(projectConfig.rawConfig, undefined, 2));
+            // write sources
+            const sources = workspace.getSources();
+            for (const source of sources) {
+                context.logger.debug(`[TEST] Writing source ${source.absoluteFilePath}`);
+                const sourceContents = await readFile(source.absoluteFilePath);
+                const relativeLocation = relative(workspace.absoluteFilepath, source.absoluteFilePath);
```

Comment:
What does `relative` return if the source absolute filepath is outside of the fern folder?

Reply:
I wonder if we need to check the returned result to make sure lines 158-162 dont throw

Reply:
I've wrapped it all in a try catch